library/src/main/java/com/datatorrent/lib/io/jms/JMSTransactionableStore.java

```diff
@@ -33,7 +33,10 @@
 
 /**
  * This transactionable store commits the messages sent within a window along with the windowId of the completed window
- * to JMS. This store ensures that the JMS output operator is capable of outputting data to JMS exactly once.
+ * to JMS. WindowIds will be sent to a subject specific meta-queue with the name of the form '{subject}.metadata'. Physical
+ * creation of this meta-queue is required by the JMS provider.
```

Comment:
Should have been '... required in the JMS provider'. 
What do you think about: "It is the responsibility of the user to create the meta-data queue in the JMS provider."